### DISCUSSION TITLE
INSERT INTO SELECT 写入提示成功 但是查不到

### DISCUSSION THREAD

**Discussion by Hby118** [apache/doris#21280] [RFC]
INSERT INTO table SELECT  ...
执行完成功提示写入了,而且有一定数据量

但是SELECT COUNT(1) 结果一直是0

重复过几次了,偶尔能出现成功的.这是为啥,今天才出现这个现象.

**Comment by yagagagaga** [comment]
What's your Doris's version and how to reproduce?

**Reply by Hby118** [reply]
昨天是1.1.3  现在升级到了1.2.4了 我再观察观察把
